backend/src/main/java/de/uslu/Travel/TravelController.java

```diff
@@ -0,0 +1,27 @@
+package de.uslu.Travel;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/travel")
```

Comment:
```suggestion
@RequestMapping("/api/travel")
```

Consider adding /api to your backend-requests to avoid conflicts with your Frontend-Router (url backend: /api/travel, url for frontend: /travel)